de/ospo.md

```diff
@@ -28,3 +28,24 @@ Das OSPO bietet Beratung bei neuen Softwarelösungen (Beschaffungen, Ablöse, kl
 * Mitarbeit bei Fehlanzeigen, wenn es keine praktikable Open Source Lösung gibt.
 
 Das OSPO bietet nur beratende Mithilfe, keine Ausschreibungsdurchführung oder Make-Buy-Use-Compose Analysen (MBUC) an.
+
+
+## Team
+
+Neben zahlreichen Entwickler\*innen der städtischen Open Source Community, die Ideen, Leidenschaft und vor Public Code einbringen, kümmern sich um Open Source:
```

Comment:
Maybe add https://github.com/orgs/it-at-m/people , too?

Reply:
I've thought about that too, but on https://github.com/orgs/it-at-m/people only 15 people are shown for non-groupmembers, because most of them haven't set their membership to public. But we are almost 90 people